services/user.service.js

```diff
@@ -53,39 +62,43 @@ const createUser = async (body) => {
     createdAt,
     updatedAt
   );
-
-  if (newUser) {
-    return {
-      status: 201,
-      message: `User: ${username} is successfully created`,
-    };
-  }
+ return { status: 201, message: newUser }; 
+ 
 };
 
-const getUserByUsername = async (usernameParamData) => {
+
+const getUserByUsername = async (usernameParamData,fetchPassword) => {
```

Comment:
fetchPassword parameter should be false by default.

Reply:
Sure, I have changed that.